src/index.html

```diff
@@ -30,22 +30,22 @@
             <a href="" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Смарфоны и телефоны</a>
+            <a href="" class="nav__link">Smartphones & phones</a>
```

Comment:
imho, for English we can just put Smartphones.
```suggestion
            <a href="" class="nav__link">Smartphones</a>
```